azure-mgmt-containerservice/src/main/java/com/azure/management/containerservice/AgentPoolAvailableVersionsPropertiesAgentPoolVersionsItem.java

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.management.containerservice;
```

Comment:
same here, should we change namespace to kubernets service?

Reply:
containerservice here, I think we couldn't change. https://github.com/azure/azure-rest-api-specs/blob/1c3a60403422ca8c73df7eb7c65ae2a3c90bf152/specification/containerservice/resource-manager/readme.md#L1

Reply:
@yungezz this namespace is defined in readme.md by the service team when write the Swagger spec. All the SDK will have the similar namespace. 
 
FYI: 
       `ContainerService`: AKS
       `ContainerInstance`: ACI
       `ContainerRegistry`: ACR